core/src/main/resources/reference.conf

```diff
@@ -461,13 +461,21 @@ languages {
         }
       }
       "biscayne" {
-        # WDL biscayne is our in-progress name for what will (probably) become WDL 1.1
+        # WDL biscayne is our in-progress name for what will become WDL 1.1
         language-factory = "languages.wdl.biscayne.WdlBiscayneLanguageFactory"
         config {
           strict-validation: true
           enabled: true
         }
       }
+      "cascades" {
+        # WDL cascades is our in-progress name for what will (probably) become WDL 2.0
+        language-factory = "languages.wdl.cascades.WdlCascadesLanguageFactory"
+        config {
+          strict-validation: true
+          enabled: true
+        }
+      }
```

Comment:
I suspect the answer is "yes" but want to confirm - this is all the config needed to support this change, right?  We don't need to adjust anything in Terra deployments?

Reply:
For the sake of completeness, filed https://github.com/broadinstitute/firecloud-develop/pull/3349. Also fixes the current `development` not having caching - not a big deal but should be consistent.